src/main/java/edu/oregonstate/cs361/battleship/ComputerAI.java

```diff
@@ -61,4 +61,27 @@ public Point fire() {
         cpuTargetFired = targetDeterminator();
         return cpuTargetFired;
     }
+
+
+    /*
+    * computer begins with random choosing location to fire at on the players's board.
+    * if the computer hits a ship w/ its fire, then, it will try to sink the ship.
+    * the computer must adhere to the player rules of battleship. the computer cannot "cheat"
+    * by seeing the location of the player's bound or ship. These are the requirements of project 4.
+    * */
+    public boolean levelOfDifficulty(boolean difficulty_hard) {
```

Comment:
I think that this is a good start to solving the difficulty problem. The existing code looks fine to me.